src/Kiota.Builder/PluginSecurityScheme.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Kiota.Builder;
+/// <summary>
+/// Supported security schemes for generating plugin auth objects
+/// </summary>
+public enum PluginSecurityScheme
```

Comment:
why not simply OpenAPISecurityScheme? In fact OAI.net [already has an enum](https://github.com/microsoft/OpenAPI.NET/blob/582e00a7c0a1e63c8ba2d3ad64ec454e8881a4fd/src/Microsoft.OpenApi/Models/SecuritySchemeType.cs#L11) we should reuse instead of duplicating here.